### PR TITLE
add awesome hosts

### PR DESCRIPTION
**Pull request by tbds**

### CODE REVIEW

**Comment by rghv** [comment]
:+1: for the awesome hosts file list

**Comment by sindresorhus** [comment]
Please read the [contributing guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [awesome manifest](https://github.com/sindresorhus/awesome/blob/master/awesome.md).

Especially this part:

> Wait at least a couple of weeks after creating a list before submitting to give it a chance to mature.

Ping me here in a few weeks and I'll reopen ;)

File: readme.md [hunk 1]
```diff
@@ -244,6 +244,7 @@ If you want to contribute, please read the [contribution guidelines](contributin
 - [Android Security](https://github.com/ashishb/android-security-awesome)
 - [Hacking](https://github.com/carpedm20/awesome-hacking)
 - [Honeypots](https://github.com/paralax/awesome-honeypots)
+- [Hosts](https://github.com/tbds/awesome-hosts)
```
Comment (sindresorhus) [inline readme.md]: `Hosts` isn't very clear. Add a succinct description:

```
- [Hosts](https://github.com/tbds/awesome-hosts) *Short description*
```

**Comment by sindresorhus** [comment]
Each items in your list could use a description.

**Pull request closed without merging**